tests/ui/privacy/private-in-public.rs

```diff
@@ -10,16 +13,25 @@ mod types {
         type Alias;
     }
 
-    pub const C: Priv = Priv; //~ ERROR private type `types::Priv` in public interface
-    pub static S: Priv = Priv; //~ ERROR private type `types::Priv` in public interface
-    pub fn f1(arg: Priv) {} //~ ERROR private type `types::Priv` in public interface
-    pub fn f2() -> Priv { panic!() } //~ ERROR private type `types::Priv` in public interface
-    pub struct S1(pub Priv); //~ ERROR private type `types::Priv` in public interface
-    pub struct S2 { pub field: Priv } //~ ERROR private type `types::Priv` in public interface
+    pub const C: Priv = Priv;
+    //~^ ERROR type `types::Priv` is more private than the item `C`
```

Comment:
Could you avoid unnecessary changes like moving `ERROR` annotations to different lines?
Makes it harder to see what changed.

Reply:
This applies to all tests, not just this one.